### PR TITLE
feat: added missing texts to docs

### PR DESCRIPTION
**Pull request by JanssenBrm**

### CODE REVIEW

**Review by Pratichhya — APPROVED** [review]

File: Applications/JupyterHub.qmd
Comment (dirkdaems) [inline Applications/JupyterHub.qmd:1]: The image of jupyterlab still contains the s3-bucket folder, while on production this has changed to 'mystorage'

Comment (Pratichhya) [reply Applications/JupyterHub.qmd:1]: Ahh sorry missed that out. I will quickly change

Comment (Pratichhya) [reply Applications/JupyterHub.qmd:1]: updated images

File: Applications/JupyterHub.qmd [hunk 1]
```diff
@@ -108,6 +116,36 @@ Working with a Notebook is pretty convenient and the supporting text should enab
 | M                 | Change cell to Markdown (text) mode |
 
 
+## Storage
+
+When you start the notebook, in the file navigation pane (the sidebar - #2) you will see three folders:
+
+* `/mystorage`
```
Comment (soxofaan) [inline Applications/JupyterHub.qmd:123]:
```suggestion
* `~/mystorage`
```

If you write it like `/foo`, it looks like a toplevel folder. I would add a `~` prefix (to indicate that it is under one's home folder),  or without anything, so just `mystorage`

File: Applications/JupyterHub.qmd [hunk 2]
```diff
@@ -108,6 +116,36 @@ Working with a Notebook is pretty convenient and the supporting text should enab
...
+This is a persistent storage with 10GB of space, which is automatically created for each user during the first login to Jupyter Hub.
+The storage is hosted in the CloudFerro cloud and can be used to save notebooks between the sessions, store the results or uploaded data files.
+This storage area is preserved when you logout and the Jupyter kernel is shut down due to inactivity.
+It will be kept for up to 15 days from your last login, and you will receive a notification to log in to Jupyter Hub to reset the timer and keep the data preserved.
+If you do not log in, then after 15 days your files will be deleted from CloudFerro cloud storage.
+
+### `/samples` and `/work`
+
+These folders are recreated with every start of the Jupyter kernel.
+The samples folder is always up-to-date with the latest version of notebooks.
```
Comment (soxofaan) [inline Applications/JupyterHub.qmd:138]:
```suggestion
The `samples` folder is always up-to-date with the latest version of notebooks.
```

File: Applications/JupyterHub.qmd [hunk 3]
```diff
@@ -108,6 +116,36 @@ Working with a Notebook is pretty convenient and the supporting text should enab
...
+If you make changes to the samples please make sure to save the updated notebook in the `/mystorage` area to make sure you can use it when you get back after period of incativity.
+
```
Comment (soxofaan) [inline Applications/JupyterHub.qmd:140]: what's the `work` folder for?

Comment (soxofaan) [reply Applications/JupyterHub.qmd:140]: and what happens with files/folders created outside of these three folders?

File: Applications/JupyterHub.qmd (hunk 1 above)
Comment (JanssenBrm) [reply Applications/JupyterHub.qmd:123]: Thanks for the suggestion @soxofaan  - I think without any symbol would be the best solution as this is the way it is shown in the UI

File: Applications/JupyterHub.qmd (hunk 3 above)
Comment (JanssenBrm) [reply Applications/JupyterHub.qmd:141]: The work folder will be removed as it has no specific use

**Pull request merged**